src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -333,17 +278,14 @@ public int compare(final ProTerritory t1, final ProTerritory t2) {
     });
 
     // Log prioritized territories
-    for (final ProTerritory patd : prioritizedTerritories) {
+    for (final ProTerritory patd : attackOptions) {
```

Comment:
Another good candidate for lambda expression replacement:
attackOptions.forEach( patd -> ProLogger.debug( ... ) );